.circleci/config.yml

```diff
@@ -6,6 +6,9 @@ executors:
       SERVER_IMAGE_NAME: decathlon/ara-server
     docker:
       - image: circleci/openjdk:8u222-stretch
+  client-builder:
+    docker:
+      - image: circleci/node:6.10
```

Comment:
I think we can take the oppportunity of this work to upgrade the node version used (I managed to work on all the 4.1.0 version with a node 10 locally without any issues)

Maybe using `circleci/node:10` ?

Reply:
When I used the latest version, I had some issues with `npm install`, so I set the version I currently have on my machine.
I can nonetheless try with this version and see if it works :)